docs/cloud/managing-airbyte-cloud/manage-schema-changes.md

```diff
@@ -29,11 +44,10 @@ To review non-breaking schema changes:
 
 ## Fix breaking schema changes
 
-:::note 
-
-Breaking changes can only occur in the **Cursor** or **Primary key** fields.
-    
-:::
+Breaking schema changes occur when:
+* The data type of a field from the source changes
+* An existing primary key is removed from the source
+* An existing cursor is removed from the source
 
 To review and fix breaking schema changes:
```

Comment:
What is the fix is the data type changes?

Reply:
Ah, I had a typo in my original question.

I meant to ask  - what should a user do when they encounter a data type change? Is the only option in that case to run a reset?